packages/sdk-codegen/src/kotlin.gen.ts

```diff
@@ -291,12 +291,20 @@ ${this.commentHeader(
   indent,
   this.methodHeaderComment(method, streamer)
 ).trimEnd()}
-${indent}${this.jvmOverloads(method)}fun ${method.name}(
+${this.deprecated(indent, method)}${indent}${this.jvmOverloads(method)}fun ${
+      method.name
+    }(
```

Comment:
This change was the source of my confusion. It's not related to deprecated method params - instead it's back-filling already existing functionality to deprecate a whole method. Typescript already had it and you added it in this PR kotlin, csharp, go, swift but not python.

When I saw `deprecated` showing up in generated code for some languages and not others I got really confused.

I'm ok with this as is, but I feel like had this bullet point in your PR description been a different PR I would have gotten through this one a lot quicker.

`adds deprecated attributes for relevant Kotlin, Swift, Go, and C# methods`